compiler/rustc_hir/src/lang_items.rs

```diff
@@ -103,13 +109,13 @@ macro_rules! language_item_table {
                 self.items[it as usize].ok_or_else(|| format!("requires `{}` lang_item", it.name()))
             }
 
+            /// Returns the [`DefId`]s of all lang items in a group.
```

Comment:
Does this include language items that aren't in the current crate?

Reply:
Given https://github.com/rust-lang/rust/pull/82641#discussion_r591063586, I would assume yes.

Reply:
Ok, can you add that as a comment?

Reply:
I can, but it seems redundant given that we say

> Defined lang items can come from the current crate or its dependencies.

at the top of the docs.

Reply:
Ah ok, I missed that comment.